gradle/publish-bom.gradle

```diff
@@ -1,7 +1,7 @@
 publishing {
     publications {
         nebulaBom(MavenPublication) {
-            nebulaDependencyManagement.fromConfigurations { configurations.compile }
+            configurations.compile
```

Comment:
beware that we haven't ironed out kinks with gradle-consistent-versions & BOM publishing yet :/

We had one PR in flight here: https://github.com/palantir/gradle-consistent-versions/pull/23, but there's also a different approach internally in the foundry-spark repo too..

Reply:
good to know, was curious to test this out. FWIW I don't know that anyone is actually consuming the tritium BOM, and ideally they wouldn't need to.